evmrpc/block_test.go

```diff
@@ -109,13 +136,9 @@ func TestGetBlockByHash(t *testing.T) {
 	privKey := hd.Secp256k1.Generate()(derivedPriv)
 	testPrivHex := hex.EncodeToString(privKey.Bytes())
 	key, _ := crypto.HexToECDSA(testPrivHex)
-	evmParams := k.GetParams(ctx)
-	evmParams.ChainConfig.CancunTime = 0 // overwrite to enable cancun
```

Comment:
is this not needed anymore?

Reply:
yeah we don't need this here; it was copied from the ante handler test which tests for different versions